package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@chainsafe/bls",
-  "version": "7.1.3",
+  "version": "7.2.0",
```

Comment:
@jeluard This branch was based off of the most recent `master` and has all the rest of the changes you made recently.  If this change gets merged we can close PR #165

Reply:
migrating to `@chainsafe/blst` 1.0.0 is a very big thing, should we make this "8.0.0" although apis are compatible? that way we can enhance/make hot fix for old versions with `7.x.x`

Reply:
I thought about that but the api of this library didnt change much so its not technically a breaking change.  Even thought the implementation changed, only the `multiplyBy` was added so from API consumer perspective so I made it a minor upgrade.  I'm not tied to that decision though.  If consensus says to make it a major I'm game.  What do you think @wemeetagain @nazarhussain @nflaig @jeluard ?

Reply:
I agree that moving to a major release would make sense if only to communicate that there was a big internal change that requires attention.

Reply:
Yes the architectural change in `blst@1.0.0` is really big, so there should be clear indication of someone using `bls` library. Upgrading to a major release with clear release notes on why it's a major release would be helpful for users.

Reply:
+1 on doing a major bump, if people are facing an issue for one reason or another due to the change of the underlying bls implementation, they can just stick with 7.x.x until those are figured out.